### ISSUE TITLE
(suggestion) User experience improvement suggestions: Bring supported boards to the README.md, use appropriate tags for positioning

### ISSUE THREAD

**Issue by vmayoral** [Xilinx/PYNQ_Composable_Pipeline#45]
This is a great piece of work @mariodruiz, congrats.

As a suggestion for improvement, I'd propose we drop a short section in the README indicating which boards are supported. It's awesome you have KV260, this should be brought straight to the README.

For clarity, this is well documented at https://pynq-composable.readthedocs.io/en/latest/getting_started.html#supported-boards but it takes a few minutes to navigate around until you find that, loosing potential user's interest. Most users will assume that the supported boards are defined in the README (currently only listing pointers for two of the supported ones). 

There are currently no tags in this repo, I'd encourage to use a few that make sense and help index/position it.

**Comment by vmayoral** [comment]
Also, you video at https://pynq-composable.readthedocs.io/en/latest/index.html is pretty good. I'd suggest bringing that upfront as well.

**Comment by mariodruiz** [comment]
Hi @vmayoral, thank you for the suggestion. The readme has this section https://github.com/Xilinx/PYNQ_Composable_Pipeline/tree/v1.0.0#rebuild-composable-pipeline-from-scratch, perhaps I can reword that section.

**Comment by vmayoral** [comment]
Ok, I see what happened in here. 
What got me confused is that I was looking at `main` branch (default one and the one used these days by open source folks for the latest tree development stuff) thinking that it contained latest development progress. I was wrong.

This indeed addresses my ask. Thanks! It might be worth considering to default the repo into your development branch,  and to hint to the stable releases somehow in the README so that folks willing to contribute can navigate around.

**Comment by mariodruiz** [comment]
Good point, the stable release is in main, the rest is development. I will add `-dev` to the development branch
